package.json

```diff
@@ -22,6 +22,7 @@
     "openapi-typescript": "^5.0.0",
     "prettier": "2.7.1"
   },
+  "prettier": {},
```

Comment:
just curious, is this necessary?

Reply:
Not explicitely. I added it only to ensure nothing else was overriding prettier config
ref: https://github.com/octokit/webhooks/pull/239